superset/db_engine_specs/base.py

```diff
@@ -271,6 +272,17 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
     run_multiple_statements_as_one = False
     custom_errors: Dict[Pattern[str], Tuple[str, SupersetErrorType]] = {}
 
+    # schema describing the parameters used to configure the DB
+    parameters_schema: Schema = None
+
+    # recommended driver name for the DB engine spec
+    drivername: str = ""
+
+    # placeholder with the SQLAlchemy URI template
+    sqlalchemy_uri_placeholder = (
+        "drivername://user:password@host:port/dbname[?key=value&key=value...]"
```

Comment:
we'll want to customize this string placeholder for BigQuery since we can do that now. :)